README.md

```diff
@@ -61,6 +61,7 @@ Validating Webhook:
 
 - In a `AWSMachineDeployment` resource, it validates the Machine Deployment ID is matching against `MachineDeployment` resource.
 - In a `AWSMachineDeployment` resource, on creation it validates that the `Cluster` is not deleted.
+- In an `AWSMachinedeployment` resource, it validates that the `max` number of nodes is not 0 and greater or equal to `min`.
```

Comment:
lines above we have `a` and the new entry has `an` that feels weird we should be consistent :p

Reply:
I didn't even notice. :smile: but I think `an` is correct so I'll change everything else.